### PR TITLE
feat: [CDS-36918]: env variables and service overrides

### PR DESCRIPTION
**Pull request by AshwinBhatkal-Harness**
##### Summary:

Added env variables and service overrides

##### Jira Links:

https://harness.atlassian.net/jira/software/c/projects/PIE/boards/149

##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by AshwinBhatkal-Harness** [comment]
retrigger eslint

**Review by reetika04 — APPROVED** [review]

File: src/modules/20-rbac/components/ResourcesCard/ResourcesCard.tsx [hunk 1]
```diff
@@ -82,7 +82,7 @@ const ResourcesCard: React.FC<ResourcesCardProps> = ({
                       openAddResourceModal(resourceType, Array.isArray(resourceValues) ? resourceValues : [])
                     }}
                   >
-                    {getString('rbac.resourceGroup.add')}
+                    {getString('plusAdd')}
```
Comment (vkbansal) [inline src/modules/20-rbac/components/ResourcesCard/ResourcesCard.tsx:85]: this should be `common.plusAdd`

Comment (AshwinBhatkal-Harness) [reply src/modules/20-rbac/components/ResourcesCard/ResourcesCard.tsx:85]: This is not present in common strings. Shall I move it?

Comment (vkbansal) [reply src/modules/20-rbac/components/ResourcesCard/ResourcesCard.tsx:85]: Ignore

**Review by vkbansal — APPROVED** [review]

**Pull request merged**